server/initializers/checker.ts

```diff
@@ -64,6 +67,16 @@ async function checkFFmpeg (CONFIG: { TRANSCODING: { ENABLED: boolean } }) {
       throw new Error('Unavailable encode codec ' + codec + ' in FFmpeg')
     }
   }
+
+  const optionalEncoders = [ 'libfdk_aac' ]
```

Comment:
I would prefer a dedicated function that fetch available encoders, create an hashmap and cache it. This function would called by checkffmpeg. The main goal is to avoid the for block each time we want to check an encoder is supported + improve efficiency with a hashmap (~ O(1))